src/feedinlib/cds_request_tools.py

```diff
@@ -83,7 +85,10 @@ def _get_cds_data(
     ), "Need to specify at least 'variable', 'year' and 'month'"
 
     # Send the data request to the server
-    result = cds_client.retrieve(dataset_name, request,)
+    result = cds_client.retrieve(
+        dataset_name,
+        request,
+    )
```

Comment:
This line isn't to long so it' probably only broken by `black` because of the trailing comma in the argument list. Remove the comma or use the `--skip-magic-trailing-comma` / `-C` of `black>=21.4b0`. Again, more of a note to myself than something stopping this merge.